demos/twa-webview-fallback/src/main/java/com/google/browser/examples/twawebviewfallback/WebViewFallbackActivity.java

```diff
@@ -30,15 +31,30 @@
 public class WebViewFallbackActivity extends AppCompatActivity {
     public static final String KEY_LAUNCH_URI =
             "com.google.browser.examples.twawebviewfallback.WebViewFallbackActivity.KEY_LAUNCH_URL";
+    public static final String NAVIGATION_BAR_COLOR =
+            "com.google.browser.examples.twawebviewfallback.WebViewFallbackActivity" +
+                    ".NAVIGATION_BAR_COLOR";
+    public static final String STATUS_BAR_COLOR =
+            "com.google.browser.examples.twawebviewfallback.WebViewFallbackActivity" +
+                    ".STATUS_BAR_COLOR";
+
 
     private Uri mLaunchUrl;
+    private int mStatusBarColor;
 
+    @SuppressLint("SetJavaScriptEnabled")
```

Comment:
What's this about?

Reply:
Android Studio linter alerts when we do `WebViewSettings#setJavaScritpEnabled(true)`. But this is required in this case.